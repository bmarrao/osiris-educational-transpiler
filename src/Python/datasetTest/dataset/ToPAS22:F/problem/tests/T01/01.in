5
3
4 -1 6
3
4 5 6
3
4 -1 4
5
-1 -1 4 6 -1
3
-1 -1 -1
