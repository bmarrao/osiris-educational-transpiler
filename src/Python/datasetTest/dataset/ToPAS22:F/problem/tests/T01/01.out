Possivel
Possivel
Impossivel
Possivel
Possivel
